### ISSUE TITLE
Supported parameters NCBI BLAST API

### ISSUE THREAD

**Issue by comatrion** [biogo/ncbi#14]
There seem to be some parameters supported in the biogo blast interface e.g. `PERC_IDENT` that are not supported (or 'should not be relied upon') according to the NCBI's documentation https://ncbi.github.io/blast-cloud/dev/api.html. Is this the correct documentation relevant to the biogo interface or has there been some change?

**Comment by kortschak** [comment]
Thank you for the report. bíogo development is currently unfunded and so fixes and improvements are provided on a best effort basis.

Having said that, a way forward for this would be to take this opportunity to migrate from XML serialisation, add in the additional blast result records components that were added a while back and so  leave v1, moving to biogo/ncbi v2. I'd be very happy for the XML stuff to go away. I will not be likely to get to do this anytime soon, but will review PRs.

**Comment by kortschak** [comment]
@peterjc Do you have a pointer to where the BLAST+ "Single-file BLAST JSON" (`--outfmt=15`) spec lives?

**Comment by peterjc** [comment]
No, but it would be worth emailing the blast team and asking if you can't find this in the online docs.

**Comment by kortschak** [comment]
The only data schema descriptions are XSD and DTD [here](https://www.ncbi.nlm.nih.gov/data_specs/). They have nothing nice, but I imagine that the JSON will conform to the XML as much as it in principle can.

**Comment by kortschak** [comment]
From NLM support:
>We do not have JSON specs. Checking with our developers, the JSON output is based on the XML output and for that you already know where the DTDs are.

>Our C++ document contains a json wrapper section, which may be of some additional help:
https://ncbi.github.io/cxx-toolkit/pages/ch_jsonwrapp
